hugegraph-api/src/main/java/com/baidu/hugegraph/config/ServerOptions.java

```diff
@@ -250,4 +250,12 @@ public static synchronized ServerOptions instance() {
                     null,
                     "hugegraph"
             );
+
+    public static final ConfigOption<Boolean>  DYNAMIC_CREATE_GRAPH =
+            new ConfigOption<>(
+                    "dynamic_create_graph",
+                    "Whether to create graph dynamically",
+                    disallowEmpty(),
+                    true
```

Comment:
Thank you for your contribution.
Not sure in what scenario the option needs to be turned off?

Reply:
Hi @javeme , thanks for your reply, I am worried that my online stable graph instance will be deleted by misoperation.

Reply:
Get it, sounds make sense